update_handlers.go

```diff
@@ -0,0 +1,131 @@
+package handlers
+
+import (
+    "os"
+    "time"
+    "github.com/gofiber/fiber/v2"
+    "github.com/golang-jwt/jwt/v4"
+    "github.com/sandbox-science/online-learning-platform/configs/database"
+    "github.com/sandbox-science/online-learning-platform/internal/entity"
+    "github.com/sandbox-science/online-learning-platform/internal/utils"
+)
+
+// UpdateUsername updates a user's username
+func UpdateUsername(c *fiber.Ctx) error {
+    var data map[string]string
+
+    if err := c.BodyParser(&data); err != nil {
+        return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"message": "Invalid input"})
+    }
+
+    userID := data["user_id"]
+    username := data["username"]
+
+    var user entity.Account
+    if err := database.DB.First(&user, userID).Error; err != nil {
+        return c.Status(fiber.StatusNotFound).JSON(fiber.Map{"message": "User not found"})
+    }
+
+    user.Username = username
+    if err := database.DB.Save(&user).Error; err != nil {
+        return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"message": "Couldn't update username"})
+    }
+
+    return c.JSON(fiber.Map{"message": "Username updated successfully"})
+}
+
+// UpdateEmail updates a user's email
+func UpdateEmail(c *fiber.Ctx) error {
+    var data map[string]string
+
+    if err := c.BodyParser(&data); err != nil {
+        return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"message": "Invalid input"})
+    }
+
+    userID := data["user_id"]
+    email := data["email"]
+    confirmEmail := data["confirm_email"]
+
+    if email != confirmEmail {
+        return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"message": "Emails do not match"})
+    }
+
+    var user entity.Account
+    if err := database.DB.First(&user, userID).Error; err != nil {
```

Comment:
nit: We may probably want to have this condition before checking if the email and confirmEmail are matching. This would avoid having unnecessary running conditions if the user does not exist in the database.
In other words, move lines 53 to 56 above ``if email != confirmEmail {``